### PR TITLE
Add responsive layouts for mobile, tablet, desktop, and large format 

### PR DESCRIPTION
**Pull request by Garrett-Iannuzzi**
**Fixes issues:** #4  #5 and #7 by adding responsive layouts for mobile, tablet, desktop, and large formats. Media queries have been applied at three different breakpoints that are assigned to variables at the top of the media-queries SCSS file. The fonts have been changed to ems for accessibility purposes as well. This will also need to be done in the main css file.  

If you would like to clone down this branch you can test how the layout changes. 
**Key areas of change in the layout:**
- The display settings in the header, flex has been added .
- Font sizes for the h1 header (changed to ems), date, welcome message, and icons.
- Spacing of the icons and brightness bar.

@JamesRexMiller4 @RayRedGoose let me know what input you have so I can make appropriate changes

**This would close issues:** #4  #5 and #7

**Screenshot Mobile:**
Screen Shot 2019-10-20 at 7 33 52 PM

**Screenshot Tablet:**
Screen Shot 2019-10-20 at 7 35 18 PM

**Screenshot Desktop:**
Screen Shot 2019-10-20 at 7 36 15 PM

### CODE REVIEW

File: src/css/media-queries.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    ul {
+      width: 50%;
+    }
+  }
+}
+
+@media screen and (min-width: $break-largeSize){
+
+  .header {
+    // background-color: lavender;
```
Comment (JamesRexMiller4) [inline src/css/media-queries.scss]: Got an unused commented out section. If you can remove this, I will go ahead with the approval

**Comment by JamesRexMiller4** [comment]
Really descriptive PR, screenshots look good. Played around on the live server and checked out the each of the different formats (mobile, tablet, desktop). Mobile looks great as is. Tablet I could nitpick and say that there is excessive spacing in between some of the widgets. But overall solid code, and thanks for tackling this. Responsiveness is always a pain. Found one bit of code that was commented out, if you could remove it I will go ahead and approve

File: src/css/media-queries.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    .header-date {
+      font-size: 1em;
+    }
+    .switch__dropdown {
+      display: flex;
+      margin-bottom: 20px;
+      justify-content: space-between;
+    }
+    .switch {
```
Comment (RayRedGoose) [inline src/css/media-queries.scss]: Everything looks good! Awesome that you change sizes into more dynamic.  I found  this part of code that doesn't look good, maybe you can use other way for making spaces.

File: src/css/media-queries.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      margin-right: 135px;
+    }
```
Comment (RayRedGoose) [inline src/css/media-queries.scss]: Everything looks good! Awesome that you change sizes into more dynamic. I found this part of code that doesn't look good, maybe you can use other way for making spaces.

**Pull request merged**